### PR TITLE
Updates to the release checklists

### PR DESCRIPTION
**Pull request by NickDraper**
**Description of work.**

Dev doc updates to the release checklists to bring them up to date with current practices

**To test:**

1. A quick once over visually of the changes is all that is needed.

*This does not require release notes* because it is dev docs only

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: dev-docs/source/PatchReleaseChecklist.rst [hunk 1]
```diff
@@ -14,13 +14,12 @@ Request
 #######
 
 *  Anyone may request a patch release, but that request must be intially
-   approved by Project manager (Nick) or one of the team leaders (Pete
-   or Mattieu).
+   approved ont of the Local PMs.
```
Comment (martyngigg) [inline dev-docs/source/PatchReleaseChecklist.rst:17]:
```suggestion
   approved by one of the local PMs.
```

File: dev-docs/source/PatchReleaseChecklist.rst [hunk 2]
```diff
@@ -14,13 +14,12 @@ Request
...
 
 Authorisation
 #############
 
-*  The Project Manager and Team leaders must meet to authorise the patch
+*  The Project Managent Team must meet to authorise the patch
```
Comment (martyngigg) [inline dev-docs/source/PatchReleaseChecklist.rst:22]:
```suggestion
*  The Project Management Team must meet to authorise the patch
```

File: dev-docs/source/PatchReleaseChecklist.rst [hunk 3]
```diff
@@ -141,11 +140,10 @@ Once the testing has passed:
   :ref:`release checklist <ReleaseChecklist>`.
 * Send an email, including the text of the release notes, to the
   following lists
-* ``mantid-announce@mantidproject.org``
-* ``mantid-developers@mantidproject.org``
 * ``nobugs@nobugsconference.org``
 * ``news@neutronsources.org``
 * ``neutron@neutronsources.org``
+* Also post the contents of the message on General on Slack
```
Comment (martyngigg) [inline dev-docs/source/PatchReleaseChecklist.rst:146]: I think we have been using #Announcements for this?

File: dev-docs/source/ReleaseChecklist.rst [hunk 4]
```diff
@@ -14,23 +14,27 @@ request or perform a patch release look at the
...
-*  Send an email to Mantid-developers reminding developers of the
+*  Post on the General slack channel reminding developers of the
    impending release and stating that they have only 5 days left before
    the code freeze.
 *  Send an email to beta test users explaining the dates for the
    testing, and stating they will have more detail on the start of the
-   first day.
-*  Invite staff to the release presentation
+   first day (cc the Local PMs so they can organise a similar message at their facilities).
+*  Before the code freeze is in place the PM and TSC need to agree the maintenace tasks for the next release period.
```
Comment (martyngigg) [inline dev-docs/source/ReleaseChecklist.rst:37]:
```suggestion
*  Before the code freeze is in place the PM and TSC need to agree the maintenance tasks for the next release period.
```

File: dev-docs/source/ReleaseChecklist.rst [hunk 5]
```diff
@@ -41,48 +45,73 @@ Documentation.
 Code Freeze
 -----------
 
-*  Send an email to Mantid-developers asking everyone to ensure they
-   have closed their tickets, stating the code freeze is in place, and
-   warning developers that non-blocker tickets will be moved from this
-   release on Monday.
+*  Post on the General slack channel asking everyone to ensure they
+   have move any incomplete issues, stating the code freeze is in place, and
```
Comment (martyngigg) [inline dev-docs/source/ReleaseChecklist.rst:49]:
```suggestion
   have moved any incomplete issues to the next milestone, stating the code freeze is in place, and
```

File: dev-docs/source/ReleaseChecklist.rst [hunk 6]
```diff
@@ -41,48 +45,73 @@ Documentation.
...
+   warning developers that non-blocker tickets will be moved from the
```
Comment (martyngigg) [inline dev-docs/source/ReleaseChecklist.rst:50]:
```suggestion
   warning developers that non-blocker issues will be moved from the
```

File: dev-docs/source/ReleaseChecklist.rst [hunk 7]
```diff
@@ -41,48 +45,73 @@ Documentation.
...
-----------------------------------------------
+Clearing the project board
+--------------------------
+
+* Go through the Zenhub project board for the release milestone (not the sprint milestone), ensuring that:
+
+ *  All issues are intended for the release.
+ *  Any new issues are triaged on a daily basis, and allocated to staff.
+ *  Issues that are not important for the release should be moved to a more appropriate milestone.
+    Don't leave anything in the release milestone that is not definitely for that release.
```
Comment (martyngigg) [inline dev-docs/source/ReleaseChecklist.rst:68]: :+1:

File: dev-docs/source/ReleaseChecklist.rst [hunk 8]
```diff
@@ -179,12 +227,12 @@ Finalise
 
 * Send an email, including the text of the release notes, to the
   following lists
-* ``mantid-announce@mantidproject.org``
 * ``nobugs@nobugsconference.org``
 * ``news@neutronsources.org``
 * ``neutron@neutronsources.org``
+* Also post the contents of the message on General on Slack
```
Comment (martyngigg) [inline dev-docs/source/ReleaseChecklist.rst:233]: I think we have been using Annoucements for this too.

File: dev-docs/source/ReleaseChecklist.rst [hunk 9]
```diff
@@ -179,12 +227,12 @@ Finalise
...
 * Create a new item on the forum news
-* Close the release milestone in the issue tracker
+* Close the release milestone in the github
```
Comment (martyngigg) [inline dev-docs/source/ReleaseChecklist.rst:235]:
```suggestion
* Close the release milestone on github
```

**Review by martyngigg — CHANGES_REQUESTED** [review]
Looks good. Now matches what we do these days.

Just a few comments that are most small spelling issues.

File: dev-docs/source/PatchReleaseChecklist.rst (hunk 3 above)
Comment (NickDraper) [reply dev-docs/source/PatchReleaseChecklist.rst]: Good point I've changed this

File: dev-docs/source/ReleaseChecklist.rst (hunk 8 above)
Comment (NickDraper) [reply dev-docs/source/ReleaseChecklist.rst]: Good point I've changed this

**Review by martyngigg — APPROVED** [review]

**Pull request merged**